rest-api/config/config_test.json

```diff
@@ -7,10 +7,17 @@
   "user_info": {
     "pmi-hpo-dev@appspot.gserviceaccount.com": {
       "roles": ["healthpro"],
-      "comment": "for HealthPro testing"
+      "comment": "for HealthPro testing",
+      "whitelisted_appids": ["pmi-hpo-dev"],
     },
     "vibrent-drc-qa@vibrent-drc-qa-165017.iam.gserviceaccount.com": {
       "roles": ["ptc"]
+      "whitelisted_ip_ranges": {
+        "ip4": ["34.206.2.13",
```

Comment:
For my understanding: what do these IP addresses correspond to? Are they stable? How would one regenerate this list?

Reply:
They are IP addresses Vibrent sent us. They are supposed to be stable. If they change, Vibrent would have to tell us.